lib/css/exports/_stacks-constants-colors.less

```diff
@@ -532,37 +532,36 @@
-    --theme-primary-150: .native-darken-desaturate(theme-primary-color, 22, 40)[];
-    --theme-primary-200: .native-darken-desaturate(theme-primary-color, 15, 25)[];
-    --theme-primary-300: .native-darken-desaturate(theme-primary-color, 8, 15)[];
-    --theme-primary-350: .native-darken-desaturate(theme-primary-color, 3, 10)[];
-    --theme-primary-400: var(--theme-primary-color);
-    --theme-primary-500: .native-tint(theme-primary-color, 11)[];
-    --theme-primary-600: .native-tint(theme-primary-color, 32)[];
-    --theme-primary-700: .native-tint(theme-primary-color, 45)[];
-    --theme-primary-800: .native-tint(theme-primary-color, 70)[];
-    --theme-primary-900: .native-tint(theme-primary-color, 80)[];
-
+    --theme-primary-025: .native-darken-desaturate(theme-primary-color, 17, 82)[];
+    --theme-primary-050: .native-darken-desaturate(theme-primary-color, 15, 47)[];
+    --theme-primary-075: .native-darken-desaturate(theme-primary-color, 13, 39)[];
+    --theme-primary-100: .native-darken-desaturate(theme-primary-color, 11, 32)[];
+    --theme-primary-150: .native-darken-desaturate(theme-primary-color, 9, 23)[];
+    --theme-primary-200: .native-darken-desaturate(theme-primary-color, 7, 15)[];
+    --theme-primary-300: .native-darken-desaturate(theme-primary-color, 5, 7)[];
+    --theme-primary-350: var(--theme-primary-color);
+    --theme-primary-400: .native-tint(theme-primary-color, 21)[];
+    --theme-primary-500: .native-tint(theme-primary-color, 36)[];
+    --theme-primary-600: .native-tint(theme-primary-color, 51)[];
+    --theme-primary-700: .native-tint(theme-primary-color, 66)[];
+    --theme-primary-800: .native-tint(theme-primary-color, 81)[];
+    --theme-primary-900: .native-tint(theme-primary-color, 96)[];
 
     // Secondary Theming
-    --theme-secondary-025: .native-darken-desaturate(theme-secondary-color, 33, 80)[];
-    --theme-secondary-050: .native-darken-desaturate(theme-secondary-color, 31, 72)[];
-    --theme-secondary-075: .native-darken-desaturate(theme-secondary-color, 29, 63)[];
-    --theme-secondary-100: .native-darken-desaturate(theme-secondary-color, 27, 55)[];
-    --theme-secondary-150: .native-darken-desaturate(theme-secondary-color, 22, 40)[];
-    --theme-secondary-200: .native-darken-desaturate(theme-secondary-color, 15, 25)[];
-    --theme-secondary-300: .native-darken-desaturate(theme-secondary-color, 8, 15)[];
-    --theme-secondary-350: .native-darken-desaturate(theme-secondary-color, 3, 10)[];
-    --theme-secondary-400: var(--theme-secondary-color);
-    --theme-secondary-500: .native-tint(theme-secondary-color, 11)[];
-    --theme-secondary-600: .native-tint(theme-secondary-color, 32)[];
-    --theme-secondary-700: .native-tint(theme-secondary-color, 45)[];
-    --theme-secondary-800: .native-tint(theme-secondary-color, 70)[];
-    --theme-secondary-900: .native-tint(theme-secondary-color, 80)[];
+    --theme-secondary-025: .native-darken-desaturate(theme-secondary-color, 17, 82)[];
```

Comment:
I don't love the big visual jump between `025` and `050`, but no one is ever practically using those colors together like we display them on the colors page. Contrast is king, not a smooth gradient.

Reply:
For _some_ colors, `025` winds up gray though.

![20210723-111004](https://user-images.githubusercontent.com/647177/126802545-4c0cbf57-715a-47bd-90fe-c1155618f9d8.png)

Reply:
Yeah, I pretty massively desaturated those since `025` is only used in the context of the highlighted user card, which was just way too little contrast and saturated at `050`. Happy to revisit this gradient / curve but want to improve things in dark mode in production as soon as possible.

Reply:
That's fair